monkestation/code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -0,0 +1,45 @@
+//////////////////////////////////////////////
+//                                          //
+//           MIMICS (GHOST)                 //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/from_ghosts/mimic
+	name = "Mimic"
+	antag_datum = /datum/antagonist/mimic
+	antag_flag = "Mimic"
+	antag_flag_override = ROLE_ALIEN
+	enemy_roles = list("AI", "Security Officer", "Detective", "Warden", "Head of Security", "Captain")
+	required_enemies = list(4,3,2,2,1,1,1,1,0,0)
```

Comment:
Change it to be 3 down the line

Reply:
Im assuming you mean like replace more with 3's, if not lmk

Reply:
list(3,3,3,3,3,3,3,3,3,3)